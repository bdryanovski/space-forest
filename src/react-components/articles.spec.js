import React from 'react'
import renderer from 'react-test-renderer'
import Article from './article'

describe('Article', () => {
  it('should match default snapshot', () => {
    const article = renderer.create(<Article>Hello</Article>)
    expect(article.toJSON()).toMatchSnapshot()
  })
})
