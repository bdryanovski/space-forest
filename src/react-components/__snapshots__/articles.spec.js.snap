// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article should match default snapshot 1`] = `
<div
  className="article"
>
  Hello
</div>
`;
